dockerfiles/Dockerfile

```diff
@@ -7,7 +7,15 @@ FROM python:3.8-slim
 COPY requirements.txt .
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-    build-essential default-libmysqlclient-dev netcat vim-tiny jq python3-dev git supervisor curl && \
+    build-essential \
+    default-libmysqlclient-dev \
+    netcat-openbsd \
+    vim-tiny \
```

Comment:
We probably could remove vim-tiny but maybe that's a separate issue

Reply:
Let's leave that as a separate issue.